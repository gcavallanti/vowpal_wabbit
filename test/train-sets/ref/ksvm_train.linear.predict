0.000000
0.533802
0.024339
-0.411951
-0.492665
-0.584747
-0.417858
-0.521428
-0.503527
-0.582914
-0.463158
-0.521759
-0.555031
-0.629764
-0.703308
-0.450767
-0.406695
-0.201802
-0.561376
-0.389729
-0.553136
-0.451235
-0.305807
-0.422035
-0.127303
-0.495621
-0.401073
-0.639339
-0.422899
-0.120010
-0.497807
-0.356543
-0.458425
-0.468078
-0.509064
-0.400359
-0.497955
-0.311618
-0.385408
0.049903
-0.353206
-0.368817
-0.056989
-0.383142
-0.690105
-0.617878
-0.146218
-0.501121
-0.459273
-0.260637
-0.564482
-0.303310
-0.414079
-0.290160
-0.376874
0.309951
-0.309206
-0.556519
-0.453607
-0.266231
-0.412704
-0.078820
-0.033614
-0.617392
-0.188757
-0.112526
-0.312238
-0.273125
-0.563033
-0.387907
-0.051699
-0.205813
-0.341185
-0.660809
0.102875
-0.195551
0.030161
-0.527664
-0.392530
-0.421752
-0.455696
-0.319576
-0.066873
-0.262962
-0.684682
-0.411370
-0.589703
-0.075548
-0.115878
-0.589044
-0.384284
-0.419465
0.440885
-0.310705
0.311893
-0.572666
-0.383447
-0.069443
-0.499425
-0.378975
-0.192019
-0.383127
-0.103490
-0.347130
-0.309629
-0.211497
-0.362473
-0.560248
-0.489264
-0.113481
-0.144652
-0.062828
-0.162181
-0.492240
0.198696
-0.449171
-0.410364
-0.273270
-0.037193
-0.289555
-0.241926
-0.141993
-0.150995
-0.498210
0.491776
0.258555
-0.080803
0.203093
-0.488751
-0.262879
0.026444
-0.025360
-0.029724
-0.312514
-0.295672
-0.759902
-0.194018
-0.041625
0.802207
-0.431107
-0.485974
-0.489144
0.566653
0.741802
-0.053693
0.323181
-0.255488
0.728841
-0.839170
-0.384707
0.206496
-0.297474
0.588321
0.094686
-0.550130
0.077405
0.709431
-0.072283
0.783521
-0.165165
0.601848
-0.408416
-0.228791
0.170820
-0.272027
-0.134574
0.586603
-0.338270
-0.339509
0.539590
0.493735
-0.708862
-0.334012
0.148694
-0.468877
0.008552
-0.706263
0.924668
0.261432
0.736790
0.084991
0.235927
0.153234
-0.374135
-0.250126
-0.259805
0.259987
-0.490108
-0.405003
0.386986
0.674371
0.024151
-0.332930
-0.492280
-0.373955
-0.033945
0.157177
-0.451866
-0.170447
-0.141979
-0.093816
0.012154
0.232594
-0.637748
-0.580520
-0.668805
0.563294
-0.043023
-0.034596
0.025638
-0.064474
-0.788022
-0.285406
0.104403
1.371135
-0.630095
-0.426191
-0.298773
-0.117557
-0.070479
0.177217
-0.487927
-0.492645
-0.366809
0.043866
-0.286434
1.586725
0.199918
-0.391880
-0.700873
-0.693537
-0.744741
-0.760282
-0.024022
0.114492
-0.686014
-0.480270
0.309115
1.041666
0.105214
0.199556
-0.261652
-0.805420
0.696850
-0.073834
0.453354
0.556723
-0.503148
-0.408042
0.827830
