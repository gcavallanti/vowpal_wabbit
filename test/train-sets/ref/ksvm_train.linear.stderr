using l2 regularization = 1
Lambda = 1
Kernel = linear
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
predictions = linear.predict
using no cache
Reading datafile = train-sets/rcv1_smaller.dat
num sources = 1
average    since         example     example  current  current  current
loss       last          counter      weight    label  predict features
1.000000   1.000000            1         1.0   1.0000   0.0000       50
1.266901   1.533802            2         2.0  -1.0000   0.5338      103
1.036548   0.806194            4         4.0  -1.0000  -0.4120      134
0.912373   0.788199            8         8.0  -1.0000  -0.5214      145
0.897671   0.882969           16        16.0   1.0000  -0.4508       23
0.922436   0.947200           32        32.0  -1.0000  -0.3565       31
0.921296   0.920155           64        64.0  -1.0000  -0.6174       60
0.917609   0.913923          128       128.0   1.0000   0.2031      105

finished run
number of examples = 250
weighted example sum = 250
weighted label sum = -22
average loss = 0.806317
best constant = -0.088
best constant's loss = 0.992256
total feature number = 19870
Num support = 239
Number of kernel evaluations = 140880 Number of cache queries = 228024
Total loss = 201.579
Done freeing model
Done freeing kernel params
Done with finish 
